user_reservation.py

```diff
@@ -0,0 +1,299 @@
+import json
+from datetime import datetime
+
+
+def user_reservation_module():
+    print("Za chwilę zaprezentuję listę dostepnych biurek.")
+
+    def beginning_date():
+        while True:
+            print("Od kiedy chcesz wynająć stanowisko?")
+            current_date = datetime.today().date()
+            input_beginning_date = input("Wprowadź datę w formacie YYYY-MM-DD: ")
+            try:
+                user_beginning_date = datetime.strptime(input_beginning_date, "%Y-%m-%d").date()
+                if user_beginning_date >= current_date:
+                    user_beginning_date_str = user_beginning_date.strftime("%Y-%m-%d")
+                    return user_beginning_date_str, user_beginning_date
+                else:
+                    print("Data powinna być późniejsza niż dzisiejsza. Spróbuj ponownie.")
+            except ValueError:
+                print("Wprowadź odpowiednią datę w formacie YYYY-MM-DD")
+    user_beginning_date_str, user_beginning_date = beginning_date()
+
+
+    def end_date(user_beginning_date):
+        while True:
+            print("Do kiedy chcesz wynająć stanowisko (wprowadzony dzień włącznie)?")
+            input_end_date = input("Wprowadź datę w formacie YYYY-MM-DD: ")
+            try:
+                user_end_date = datetime.strptime(input_end_date, "%Y-%m-%d").date()
+                if user_end_date >= user_beginning_date:
+                    user_end_date_str = user_end_date.strftime("%Y-%m-%d")
+                    counter = user_end_date - user_beginning_date
+                    days_counter = counter.days + 1
+                    print(f"Wybrano biurko w przedziale czasowym od {user_beginning_date} do {user_end_date}.")
+                    print(f"Całkowita liczba dni wynosi: {days_counter}")
+                    return user_end_date_str, user_end_date, days_counter
+                else:
+                    print("Data powinna być późniejsza niż data początkowa. Spróbuj ponownie.")
+            except ValueError:
+                print("Wprowadź odpowiednią datę w formacie YYYY-MM-DD")
+    user_end_date_str, user_end_date, days_counter = end_date(user_beginning_date)
+
+
+    def size_choosing():
+        print("Jakie biurko Cię interesuje?: ")
+        print("Biurko 1-osobowe - wprowadź cyfrę 1")
+        print("Biurko 4-osobowe - wprowadź cyfrę 4")
+        print("Biurko 10-osobowe - wprowadź cyfrę 10")
+        while True:
+            user_choice_size = input("Wprowadź swój wybór: ")
+            if user_choice_size.isdigit():
+                if int(user_choice_size) == 1:
```

Comment:
od lini 53 do 61 mozesz to zastapic jednym ifem i jednym returnem zmiast 4
```suggestion
                if int(user_choice_size) in ["1","4","10"]:
                     print(f"Twój wybór to biurko {user_choice_size}-osobowe.")
                     return user_choice_size 
```